pkg/apis/eventing/v1alpha1/trigger_lifecycle_test.go

```diff
@@ -43,6 +43,11 @@ var (
 		Status: corev1.ConditionTrue,
 	}
 
+	triggerConditionSubscriberReady = apis.Condition{
```

Comment:
nit: just for consistency, we should just make this variable name match. Let's do in a followup.